### DISCUSSION TITLE
Dart Version of UCUM Library

### DISCUSSION THREAD

**Discussion by Dokotela** [ucum-org/ucum#298] [Show and tell]
I work in Dart/Flutter and am involved with HL7 and FHIR, and moving into CQL. For this, I needed a UCUM package. I've built out most of the functionality here: https://github.com/fhir-fli/ucum, based on the [Ucum-Java package](https://github.com/FHIR/Ucum-java). Dart libraries are published on pub.dev for anyone to use in their applications. Before I do this, I wanted to ask if there were specific requirements in terms of Copyright and Permissions that I needed to include before I officially published it. Thanks!

**Comment by cmutel** [comment]
This is covered in https://ucum.org/license
